examples/src/odra_types.rs

```diff
@@ -0,0 +1,80 @@
+use odra::contract_env::emit_event;
+use odra::{Event, OdraType, Variable};
+
+#[odra::module]
+pub struct NestedOdraTypes {
+    operation_result: Variable<OperationResult>
```

Comment:
Add
- module in mapping
- module in mapping in mapping
- mapping in module in mapping